app/views/layouts/application.html.erb

```diff
@@ -7,15 +7,15 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%# <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %> %>
+    <%# <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
   </head>
 
   <body>
+
+<%= render "shared/navbar" %>
+<%= render "shared/flashes" %>
     <p class="notice"><%= notice %></p>
     <p class="alert"><%= alert %></p>
     <%= yield %>
   </body>
 </html>
-
```

Comment:
I moved the shared _navbar and _flashes to the top of the page. This is such that the navbar gets rendered at the top of the page, above any yield to any other HTML